add_numbers.py

```diff
@@ -0,0 +1,2 @@
+def add_numbers(a, b):
+    return a + b
\ No newline at end of file
```

Comment:
Consider adding type hints to the function parameters and return type for better readability and understanding of the function. For example: def add_numbers(a: int, b: int) -> int: